app/assets/javascripts/screen/screenController.js

```diff
@@ -28,4 +28,32 @@ angular.module('ocWebGui.screen', ['ui.router', 'ngResource', 'ocWebGui.shared.t
         });
 
         fetchData();
+        
+        // Agent status and colors classifications        
+        $scope.agents = $resource('agents.json').query(function() {
+            var agents = $scope.agents;
+            var green = 0;
+            var yellow = 0;
+            var red = 0;
+            
+            for(i=0; i< agents.length; i++) {
```

Comment:
Variable `i` is a global variable without the `var` keyword. However doing this would be nicer with JavaScript array `map` function.